README.md

```diff
@@ -209,7 +210,8 @@ Options:
 Commands:
   clean          Clean up files, only useful if you are using the default file structure
   infer          Inference
-  onnx           Export model to onnx
+  onnx
```

Comment:
did you mean to remove the onnx docstring?

Reply:
Wow, long time no see. ONNX support ended a long time ago, but I made a mistake in the order at that time.
https://github.com/voicepaw/so-vits-svc-fork/blob/df0dd614438a8fb1b4877b1f1bb5d4f0009db62c/src/so_vits_svc_fork/__main__.py#L766-L767